Readme.md

```diff
@@ -57,9 +57,22 @@ Or
     gnome tweak tool to _static_ and the _Number of Workspaces_ to the
     total number of workspaces you want to have.
 
-## Configuration options:
+## gnome-tweaks-tool (aka **Tweaks**)
+
+    sudo apt install gnome-tweak-tool
+
+Go to `Extension` an click on the gear next to `Workspace grid`.
+
+### Configuration options:
 
 - Number of rows/columns in the workspace.
+- Reative workspace switching.
```

Comment:
Reative should be Relative I guess, please fix.

Reply:
fixed